frontend/src/routes/TransactionsView.svelte

```diff
@@ -84,5 +84,10 @@
     border-left: 1px solid #ddd;
     padding: 1rem;
     overflow-y: auto;
+    @media (prefers-color-scheme: dark) {
+        background: #1a1a1a;
+        border-left: 1px solid #444;
+        color: #e0e0e0;
+    }
```

Comment:
Hi. Could we have some variables which will allow us to change these theme colors across the whole scope?


```css
:root {
  --background-dark: #1a1a1a;
  --background-light: #fff
}
.details-pane {
  background: var(--background-light);
  @media (prefers-color-scheme: dark) {
    background: var(--background-dark);
  }
}
```